### PR TITLE
fix(dialog): horizontal positioned icon

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/dialog/dialog.scss [hunk 1]
```diff
@@ -56,16 +56,27 @@
 
 	&-wrapper {
 		display: flex;
-		flex-direction: column;
 		align-items: flex-start;
-		row-gap: 24px;
+
+		.base:not(.horizontal-icon) & {
+			flex-direction: column;
+			row-gap: 24px;
```
Comment (yinonov) [inline libs/components/src/lib/dialog/dialog.scss:63]: can this be set to a derivative already? (as discussed this week regarding dense gaps)

File: libs/components/src/lib/dialog/dialog.scss [hunk 2]
```diff
@@ -56,16 +56,27 @@
...
+			.icon {
+				font-size: #{size.$vvd-size-condensed};
+			}
+		}
+
+		.base.horizontal-icon & {
+			column-gap: 8px;
```
Comment (yinonov) [inline libs/components/src/lib/dialog/dialog.scss:70]: same here

File: libs/components/src/lib/dialog/README.md [hunk 3]
```diff
@@ -43,6 +43,20 @@ Use the `icon` attribute to set the dialog's icon.
 <vwc-dialog icon="info" open></vwc-dialog>
 ```
 
+
+### horizontal-icon
+
+Use the `horizontal-icon` attribute to set the dialog's icon horizontal to dialog text
```
Comment (yinonov) [inline libs/components/src/lib/dialog/README.md:49]: thinking more on the line of `icon-placement="top | inline"`. WDYT?

File: libs/components/src/lib/dialog/dialog.template.ts [hunk 4]
```diff
@@ -89,7 +90,7 @@ export const DialogTemplate: (
 				<slot name="main">
 					<div class="main-wrapper">
 						<div class="header">
-							<div class="headline-wrapper">
+							<div class="headline-wrapper ">
```
Comment (yinonov) [inline libs/components/src/lib/dialog/dialog.template.ts:93]:
```suggestion
							<div class="headline-wrapper">
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/dialog/dialog.scss (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/dialog/dialog.scss:63]: good thinking! will change

File: libs/components/src/lib/dialog/README.md (hunk 3 above)
Comment (rachelbt) [reply libs/components/src/lib/dialog/README.md:49]: thought about it, I guess its more correct to do this.

Comment (rachelbt) [reply libs/components/src/lib/dialog/README.md]: changed

File: libs/components/src/lib/dialog/README.md [hunk 5]
```diff
@@ -43,6 +43,20 @@ Use the `icon` attribute to set the dialog's icon.
 <vwc-dialog icon="info" open></vwc-dialog>
 ```
 
+
+### Icon-placement
+
+Use the `icon-placement` attribute to set the dialog's icon horizontal to dialog text
+
+- Type: `top` | `left`
```
Comment (yinonov) [inline libs/components/src/lib/dialog/README.md:51]: Might be `right` in `rtl`. Resolve to logical?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/dialog/README.md [hunk 6]
```diff
@@ -43,6 +43,20 @@ Use the `icon` attribute to set the dialog's icon.
 <vwc-dialog icon="info" open></vwc-dialog>
 ```
 
+
+### Icon-placement
+
+Use the `icon-placement` attribute to set the dialog's icon horizontal to dialog text
```
Comment (yinonov) [inline libs/components/src/lib/dialog/README.md:49]: not clear

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/dialog/dialog.ts [hunk 7]
```diff
@@ -12,6 +12,13 @@ let dialogPolyfill: any;
 	}
 })();
 
+/**
+ * Types of icon placement
+ *
+ * @public
+ */
+type IconPlacement = 'top' | 'start';
```
[the remark this answers is not in the document]
Comment (rachelbt) [reply libs/components/src/lib/dialog/dialog.ts:20]: how about:
```suggestion
type IconPlacement = 'block-start' | 'inline-start';
```
?

File: change/@vonage-vivid-f7860a27-4ee9-4ca0-9f5c-f42d945e5647.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "dialog horizontal icon",
```
Comment (rachelbt) [inline change/@vonage-vivid-f7860a27-4ee9-4ca0-9f5c-f42d945e5647.json:3]:
```suggestion
  "comment": "dialog icon placement side option added",
```

**Review by yinonov — APPROVED** [review]

**Pull request merged**